app/views/listing/printers.php

```diff
@@ -0,0 +1,114 @@
+<?php $this->view('partials/head'); ?>
+
+<?php
+//Initialize models needed for the table
+new Machine_model;
+new Reportdata_model;
+new Printer_model;
+?>
+
+<div class="container">
+
+  <div class="row">
+
+  	<div class="col-lg-12">
+
+	<h3>Printer report <span id="total-count" class='label label-primary'>…</span></h3>
+
+	  <table class="table table-striped table-condensed table-bordered">
+
+		<thead>
+		  <tr>
+			<th data-i18n="listing.computername" data-colname='machine.computer_name'>On Computer</th>
+			<th data-i18n="serial" data-colname='reportdata.serial_number'>Computer Serial</th>
```

Comment:
If you add the data-i18n attributes, you can remove the Non-Localised strings, e.g.
```
<th data-i18n="listing.computername" data-colname='machine.computer_name'></th>
```